tools/dev-get-emails.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Returns a csv of all steward emails for all datasets in the dev environment
+
+set -eu
+
+read -rp "Enter your Broad email: " BROAD_EMAIL
+read -rp "Enter your dev email: " DEV_EMAIL
+
+gcloud auth login "${DEV_EMAIL}" 2>/dev/null
+TOKEN=$(gcloud auth print-access-token)
+gcloud auth login "${BROAD_EMAIL}" 2>/dev/null
```

Comment:
Technically for this script it doesn't, but it will be pretty confusing for developers when they move on to do other things (they need to restore their gcloud login).

Reply:
good idea!